LCM/scripts/OMS_MetaConfigHelper.py

```diff
@@ -1,12 +1,31 @@
 #!/usr/bin/python
 import os
 import sys
+import imp
+from os.path import basename, dirname, join, realpath, split
 
 conf_path = "/etc/opt/microsoft/omsagent/conf/omsadmin.conf"
 metamof_path = "/etc/opt/omi/conf/omsconfig/generated_meta_config.mof"
 agentid_path = "/etc/opt/omi/conf/omsconfig/agentid"
 omshelper_disable_path = "/etc/opt/omi/conf/omsconfig/omshelper_disable"
 
+# Redirect output to our log file
+pathToCurrentScript = realpath(__file__)
+pathToCommonScriptsFolder = dirname(pathToCurrentScript)
+fullPathDSCLogger = os.path.join(pathToCommonScriptsFolder, 'nxDSCLog.py')
+nxDSCLog = imp.load_source('nxDSCLog', fullPathDSCLogger)
+logger = nxDSCLog.ConsoleAndFileLogger()
+sys.stdout = logger
+
+def exitWithError(message, errorCode = 1):
+    errorMessage = "ERROR from InstallModule.py: " + message
```

Comment:
These should say OMS_MetaConfigHelper.py instead of InstallModule.py

Reply:
done